docs/how-to/rocks/use-flask-extension.rst

```diff
@@ -1,25 +1,53 @@
 Use the flask-framework extension
 ---------------------------------
 
-.. note::
-    The Flask extension is compatible with the ``bare`` and ``ubuntu@22.04``
-    bases.
+The ``flask-framework`` extension requires a ``requirements.txt`` file and a
+``app.py`` file that exposes ``app`` as a WSGI app. For example, ``app.py``:
```

Comment:
Nit-pick: I think `app` should be explicitly defined. Maybe something like, "...that exposes the Flask application ``app`` as a WSGI app."? (Maybe `app` doesn't refer to a Flask application in this instance.)

Reply:
Added more explanation